indexer-app/src/cmr/indexer/data/concepts/generic.clj

```diff
@@ -79,50 +79,48 @@
-             :deleted deleted
-             :gen-name gen-name
-             :gen-name-lowercase (util/safe-lowercase gen-name)
-             :gen-version gen-ver
-             :generic-type (str gen-name " " gen-ver)
-             :provider-id provider-id
-             :provider-id-lowercase (util/safe-lowercase provider-id)
-             :keyword keyword-values
-             :user-id user-id
-             :revision-date revision-date
-             :native-id native-id 
-             :native-id-lowercase native-id 
-             :associations-gzip-b64 (associations->gzip-base64-str generic-assoc-ids)}
-             configs (gen-util/only-elastic-preferences (:Indexes index-data))
-             ;; now add the configured indexes
-             doc (reduce
-                  (fn [data, config] (into data (field->index config parsed-concept)))
-                  common-doc
-                  configs)]
-             (if deleted
-               (assoc common-doc :metadata-format (name (mtype/format-key (:mime-type extra-fields)))
-                      :gen-type-lowercase (util/safe-lowercase gen-name)
-                      :long-name long-name
-                      :long-name-lowercase (util/safe-lowercase long-name))
-               doc)))))
+        index-data-file (format "schemas/%s/v%s/index.json" gen-name gen-ver)
+        index-file-raw (slurp (io/resource index-data-file))
+        index-data (json/parse-string index-file-raw true)
+        schema-keys [:LongName
+                     :Version
+                     :Description
+                     :RelatedURLs]
+        keyword-values (keyword-util/concept-keys->keyword-text
+                        parsed-concept schema-keys)
+        common-doc ;; fields common to all generic documents
+        {:concept-id concept-id
+         :revision-id revision-id
+         :deleted deleted
+         :gen-name gen-name
+         :gen-name-lowercase (util/safe-lowercase gen-name)
+         :gen-version gen-ver
+         :generic-type (str gen-name " " gen-ver)
+         :provider-id provider-id
+         :provider-id-lowercase (util/safe-lowercase provider-id)
+         :keyword keyword-values
+         :user-id user-id
+         :revision-date revision-date
+         :native-id native-id
+         :native-id-lowercase native-id
+         :associations-gzip-b64 (associations->gzip-base64-str generic-assoc-ids)}
+        configs (gen-util/only-elastic-preferences (:Indexes index-data))
+        ;; now add the configured indexes
+        doc (reduce
+             (fn [data, config] (into data (field->index config parsed-concept)))
+             common-doc
+             configs)]
+      (if deleted
+        (assoc common-doc :metadata-format (name (mtype/format-key (:mime-type extra-fields)))
+               :gen-type-lowercase (util/safe-lowercase gen-name)
+               :long-name long-name
+               :long-name-lowercase (util/safe-lowercase long-name))
+        doc)
+      ))
```

Comment:
minor: these closing parentheses shouldn't be on their own line, just let them all  sit on the end of doc

Reply:
Resolved in subsequent commit